### PR TITLE
Update code samples in 5.x docs to use const

### PR DESCRIPTION
**Pull request by RamirezAlex**
This update the code samples in 5.x alpha documentation to use the modern keyword `const` instead of `var`

Please review @crandmck @dougwilson :)

### CODE REVIEW

**Comment by crandmck** [comment]
Seems reasonable to me.  
@dougwilson @wesleytodd Do you have an opinion?

File: _includes/api/en/5x/app-listen.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The `app.listen()` method returns an [http.Server](https://nodejs.org/api/http.h
 
 ```js
 app.listen = function () {
-  var server = http.createServer(this)
+  const server = http.createServer(this)
```
Comment (dougwilson) [inline _includes/api/en/5x/app-listen.md]: This is meant to be the exact code that is in the Express source tree: https://github.com/expressjs/express/blob/5f0c829d7ca7da746ee859f13a54631000f8a9b5/lib/application.js#L602-L605

**Review by dougwilson — CHANGES_REQUESTED** [review]
I'm fine with it, though found at least one issue. There is a lot to go through, so I may have missed something. I'm not sure if this was just done with only as find and replace, or the examples were all tested to validate they are working or not still after the changes.

**Review by wesleytodd — APPROVED** [review]
Other than @dougwilson's comment, LGTM!

**Comment by crandmck** [comment]
Since he made Doug's suggested change, I'm going to merge.
@RamirezAlex sorry for the delay.

**Pull request merged**